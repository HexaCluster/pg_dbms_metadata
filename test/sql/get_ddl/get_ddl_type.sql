SELECT dbms_metadata.get_ddl('TYPE','Address','gdmmm');

SELECT dbms_metadata.get_ddl('TYPE','location','gdmmm');

-- tests for schema as NULL
SET search_path TO public, gdmmm;

SELECT dbms_metadata.get_ddl('TYPE','Address');

SELECT dbms_metadata.get_ddl('TYPE','location');
