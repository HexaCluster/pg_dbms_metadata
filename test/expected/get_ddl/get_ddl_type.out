                                          get_ddl                                           
--------------------------------------------------------------------------------------------
 CREATE TYPE gdmmm."Address" AS ( "City" character varying, loc gdmmm.location, pin bigint)
(1 row)

                                   get_ddl                                   
-----------------------------------------------------------------------------
 CREATE TYPE gdmmm.location AS ( lat double precision, lon double precision)
(1 row)

SET
                                       get_ddl                                        
--------------------------------------------------------------------------------------
 CREATE TYPE gdmmm."Address" AS ( "City" character varying, loc location, pin bigint)
(1 row)

                                   get_ddl                                   
-----------------------------------------------------------------------------
 CREATE TYPE gdmmm.location AS ( lat double precision, lon double precision)
(1 row)

